lib/src/test/java/com/auth0/jwt/JWTCreatorTest.java

```diff
@@ -530,10 +531,9 @@ public void shouldAcceptCustomMapClaimOfBasicObjectTypes() throws Exception {
 
         assertThat(jwt, is(notNullValue()));
         String[] parts = jwt.split("\\.");
-        
-        String body = new String(Base64.getUrlDecoder().decode(parts[1]), StandardCharsets.UTF_8);
-        ObjectMapper mapper = new ObjectMapper();
-        Map<String, Object> map = (Map<String, Object>) mapper.readValue(body, Map.class).get("data");
+
+        DecodedJWT decodedJWT = JWT.decode(jwt);
+        Map<String, Object> map = decodedJWT.getClaim("data").asMap();
```

Comment:
Not directly related to this change, but we should test with claims as they would be decoded normally by the library, instead of ad-hoc deserialization.